ignite/metrics/metric.py

```diff
@@ -82,7 +82,9 @@ def compute(self) -> Any:
         This is called at the end of each epoch.
 
         Returns:
-            Any: the actual quantity of interest.
+            Any: the actual quantity of interest. However, if a :class:`dict` is returned, it should contain
+                 :class:`numbers.Number` or :class:`torch.tensor` values and will flattened into
```

Comment:
Let's remove those type limitations:
```
Any: the actual quantity of interest. However, if a :class:`dict` is returned, it will flattened into  `engine.state.metrics` when :func:`~ignite.metrics.Metric.completed` is called.
```

Reply:
Done!